icu4j/main/tests/core/src/com/ibm/icu/dev/test/util/ULocaleTest.java

```diff
@@ -5154,4 +5154,138 @@ public void Test20900() {
             Assert.assertEquals(testData[row][1], loc.toLanguageTag());
         }
     }
+    @Test
+    public void TestCanonical() {
+        // Test replacement of languageAlias
+
+        // language _ variant -> language
+        Assert.assertEquals("nb",
```

Comment:
import static Assert.assertEquals so you can omit the "Assert." prefix?

Reply:
There are a lot of Assert.assertEquals in this file already and I don't want to change them in this CL.

Reply:
Also I tried but then get some cannot find method (for 2 parameters) error.